android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -814,7 +824,9 @@ public Client build(){
             if(this.MICBaseURL != null){
                client.setMICHostName(this.MICBaseURL);
             }
-
+            if (!Strings.isNullOrEmpty(this.instanceID)) {
+                client.setMICHostName(Constants.PROTOCOL_HTTPS + instanceID + Constants.HYPHEN + Constants.HOSTNAME_AUTH);
+            }
```

Comment:
Will this construct a URL that does not end with a "/"? If so, is this consistent with how the default URL is formatted?

Reply:
The SDK has logic for adding  "/" to the end of url. So all URLs will have "/" at the end.
https://github.com/Kinvey/java-library/blob/7c8e85107593870941a39c249e93464bc6238ff3/java-api-core/src/com/kinvey/java/AbstractClient.java#L115
`this.MICHostName = MICHostName.endsWith("/") ? MICHostName : MICHostName + "/"`